frontend/src/store/course/actions.ts

```diff
@@ -19,6 +23,7 @@ import {
 } from 'common/types/types';
 import { checkHasPermission } from 'helpers/helpers';
 import { notification } from 'services/services';
+import { coursesManagementActions } from 'store/actions';
```

Comment:
```suggestion
```
you don't need to change actions that are related to the `course` action. all actions should be per page